CHANGELOG.md

```diff
@@ -21,7 +21,7 @@
 
 ### Unmaintained components
 
-- `simpleprometheusreceiver`(#11133)
+- `tanzuobservabilityexporter`: remove calls to deprecated `NewProxySender` methods. (#11510)
```

Comment:
Why is this line added?

Reply:
Sorry, I mistakely add it back when I resolved the conflicts. I'll remove it

Reply:
Done